docs/netdata-cloud-onprem/getting-started.md

```diff
@@ -1,20 +1,198 @@
 # Getting started with Netdata Cloud On-Prem
+Helm charts are designed for kubernetes to run as the local equivalent of the netdata.cloud public offering. This means that no data is sent outside of your cluster. By default On-Prem installation is trying to reach for outside resources only when pulling the container images.
+
+## Requirements
+#### Install host:
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+- [Helm](https://helm.sh/docs/intro/install/) version 3.12+ with OCI Configuration (explained in installation section)
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+#### Kubernetes requirements:
+- Kubernetes cluster version 1.23+
+- Kubernetes metrics server (For autoscaling)
+- TLS certificate for Netdata Cloud On-Prem. Single endpoint is required but there is an option to split frontend, api and mqtt endpoints. Certificate must be trusted by all entities connecting to the On-Prem installation by any means.
+- Ingress controller to support HTTPS `*`
+- PostgreSQL version 13.7 `*` (Main persistent data app)
+- EMQX version 5.11 `*` (MQTT Broker that allows Agents to send messages to the On-Prem Cloud)
+- Apache Pulsar version 2.10+ `*` (Central communication hub. Applications exchange messages through Pulsar)
+- Traefik version 2.7.x `*` (Internal communication - API Gateway)
+- Elastic Search version 8.8.x `*` (Holds Feed)
+- Redis version 6.2 `*` (Cache)
+- Some form of generating imagePullSecret `*` (Our ECR repos are secured)
+- Default storage class configured and working (Persistent volumes based on SSDs are preferred)
+`*` - available in dependencies helm chart for PoC applications.
```

Comment:
This can be added as a note..

Reply:
but this is a perfect place for it, I do not get it

Reply:
I've added it right after this comment https://github.com/netdata/netdata/pull/16440/commits/0eae3020ee396f5542cc8b7d978a04412bff5722